sir_cake/store/utils.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib import messages
+from django.core.exceptions import ObjectDoesNotExist
+
+from seller.models import Item, OrderPackage
+from users.models import AnonymousUser
+from .models import Basket
+
+
+def add_one_item_basket(request, context):
+    # Anonymous user without session
+    if request.session.session_key is None:
+        return 'continue_purchase'
+
+    basket = []
+    if request.user.is_anonymous:
+        anon_user = get_user_with_saved_session(request)
+        basket = Basket.objects.filter(buyer_anon=anon_user)
+    else:
+        user = request.user
+        basket = Basket.objects.filter(buyer=user)
```

Comment:
If you only need to check number of items it's better to do `Basket.objects.count`